pinot-query-planner/src/main/java/org/apache/pinot/query/planner/stage/MailboxSendNode.java

```diff
@@ -32,17 +38,36 @@ public class MailboxSendNode extends AbstractStageNode {
   private RelDistribution.Type _exchangeType;
   @ProtoProperties
   private KeySelector<Object[], Object[]> _partitionKeySelector;
+  @ProtoProperties
+  private List<RexExpression> _collationKeys;
+  @ProtoProperties
+  private List<RelFieldCollation.Direction> _collationDirections;
+  @ProtoProperties
+  private boolean _sortOnSender;
 
   public MailboxSendNode(int stageId) {
     super(stageId);
   }
 
   public MailboxSendNode(int stageId, DataSchema dataSchema, int receiverStageId,
-      RelDistribution.Type exchangeType, @Nullable KeySelector<Object[], Object[]> partitionKeySelector) {
+      RelDistribution.Type exchangeType, @Nullable KeySelector<Object[], Object[]> partitionKeySelector,
+      @Nullable List<RelFieldCollation> fieldCollations, boolean sortOnSender) {
     super(stageId, dataSchema);
     _receiverStageId = receiverStageId;
     _exchangeType = exchangeType;
     _partitionKeySelector = partitionKeySelector;
+    if (!CollectionUtils.isEmpty(fieldCollations)) {
```

Comment:
can we add a TODO or Preconditions check to indicate that the sorting is not supported?

Reply:
done